pkg/application/application.go

```diff
@@ -0,0 +1,54 @@
+package application
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/redhat-developer/ocdev/pkg/config"
+	"github.com/redhat-developer/ocdev/pkg/occlient"
+)
+
+const defaultApplication = "app"
+
+// Create creates a new application and binds it to the current project.
+
+// If no name is provided, the application is named as in the constant
+// "defaultApplication".
+
+// If application already exists, this errors out.
+
+// If no project is set, this errors out.
+func Create(name string) error {
+	// Get current project name
+	project, err := occlient.GetCurrentProjectName()
+	if err != nil {
+		return errors.Wrap(err, "unable to get current project's name")
+	}
+
+	// Set default application name if not set
+	if len(name) == 0 {
```

Comment:
what is use case for this?
I think this should error out rather than using some default name.

Reply:
ah, ok 👍

Reply:
ok, that is more for when you are creating a component and no application is active.

ok, then let's keep it here, we will use it later